cost-analyzer/values.yaml

```diff
@@ -286,6 +286,10 @@ oidc:
   clientID: "" # application/client client_id parameter obtained from provider, used to make requests to server
   clientSecret: "" # application/client client_secret parameter obtained from provider, used to make requests to server
   secretName: "kubecost-oidc-secret" # k8s secret where clientsecret will be stored
+  customSecret:
```

Comment:
Minor comment. Thoughts on renaming this to `clientSecretName`? This way there is a clearer relationship between `clientSecret` and `clientSecretName`.

It may also be good to include an additional comment:

```yaml
clientSecretName:
  enabled: false # if true, this config takes precedence over clientSecret and specifies a k8s secret which holds the clientSecret parameter obtained from provider
  name: "" # name of the secret containing the client secret
```

Reply:
I didnt call it that because I felt like it would be confusing with the secret name field.

I can swap it to existingClientSecret if you prefer.

I don't have name as the suffix because then it becomes: clientSecretName.name which is pretty self-explanatory and felt unnecessarily duplicative. 

If you want me to remove the enabled section and only have name and then select based on if that field has a value I can do that. This felt clearer for intent however.

Reply:
Good points! I like `existingClientSecret`.